### PR TITLE
lightning v1.8.6

### PR DESCRIPTION
**Pull request by DManowitz**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

Copied dependencies from pytorch-lightning 1.8.6 commit.

### CODE REVIEW

**Comment by DManowitz** [comment]
@conda-forge-admin please rerender

**Comment by DManowitz** [comment]
Hey, @Borda, @carmocca , @jan-janssen , or @williamFalcon, can one of you please review this PR?  I didn't see any past PRs that would create the `lightning` package as opposed to the `pytorch-lightning` package, so I based my changes off of the past PRs that I did see.  I may have been wrong in my approach, though because of that.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,32 +17,32 @@ build:
...
     - pip check
   requires:
     - python >=3.7, <3.11 # TODO: this upper bound is because of PyTorch issue with dataclass
     - pip
 
 about:
-  home: https://pypi.org/project/pytorch-lightning/
-  summary: PyTorch Lightning is the lightweight PyTorch wrapper for ML researchers. Scale your models. Write less boilerplate.
+  home: https://pypi.org/project/lightning/
+  summary: Lightning is the lightweight PyTorch wrapper for ML researchers. Scale your models. Write less boilerplate.
```
Comment (Borda) [inline recipe/meta.yaml:45]:
```suggestion
  home: https://pypi.org/project/pytorch-lightning/
  summary: PyTorch Lightning is the lightweight PyTorch wrapper for ML researchers. Scale your models. Write less boilerplate.
```

**Review by Borda — CHANGES_REQUESTED** [review]
seem your changes shall be towards this package https://github.com/conda-forge/lightning-feedstock

**Pull request closed without merging**